client/state/sites/test/selectors.js

```diff
@@ -1675,6 +1676,40 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#hasJetpackSiteJetpackThemesExtendedFeatures()', () => {
```

Comment:
Yeah, as long as the selector includes the `isJetpackSite` check, we should cover that. I'll add a test.